electron/app/splash.html

```diff
@@ -6,21 +6,146 @@
             background-color: transparent ;
             overflow: hidden ;
         }
+		
+
+		.label {
```

Comment:
something's gone awry with your indentation. seems to be using tabs rather than spaces.